examples/simple/src/schema/__global__schema.ts

```diff
@@ -76,6 +76,19 @@ const glo: GlobalSchema = {
         date: TimestampType(),
         phone_number: PhoneNumberType({ nullable: true }),
         email_address: EmailType({ nullable: true }),
+        creator_id: UUIDType({
+          polymorphic: {
+            types: ["User"],
+          },
+          nullable: true,
+        }),
+        // doesn't seem like we support this...
```

Comment:
Hmmm, do we just not support this in structs or everywhere? I'm trying to think if I'm doing this anywhere. I guess not. Kinda surprised actually. I feel like I've reached most of the extreme corners of your architecture. 😛

Reply:
doesn't work for a dumb reason :( https://github.com/lolopinto/ent/blob/566a8b038762f31395401b582b673536241e0285/internal/schema/base/common.go#L113-L121